src/AzSearch.tsx

```diff
@@ -79,6 +81,15 @@ class Automagic {
             document.getElementById(htmlId)
         );
     }
+
+    public addFilterBar(htmlId: string) {
```

Comment:
I was thinking of a more complex filter "bar" with more features, but yeah right now it's just reset filters button...

Might be smart (of me) to make this specific so change component to "resetFilter" & "resetFilterContainer" for now.